variables.tf

```diff
@@ -518,3 +513,11 @@ variable "karpenter_chart_version" {
   type        = string
   default     = "v0.32.1"
 }
+
+variable "karpenter_subnet_selector_tags" {
+  description = "Subnet selector tags for Karpenter nodes"
+  type        = map(string)
+  default = {
+    "Name" = "aft-app-ap-southeast*"
+  }
```

Comment:
Let's remove the `default` and make it mandatory.

Reply:
Done

Reply:
@niroz89 if we set this as mandatory then tf will fail when assign value to `var.karpenter_nodeclasses`
https://github.com/SPHTech-Platform/platform-engg-tools/blob/1c325f30697c8ce1abe058e192a1bb760e30486e/envs/prd/github_runner_enterprise/eks_pure_karpenter_cluster/main.tf#L111